.github/actions/check-tag-version/action.yml

```diff
@@ -0,0 +1,17 @@
+---
+name: Check tag/version consistency
+description: Check that the Khiops source version matches a tag
+runs:
+  using: composite
+  steps:
+    - name: Check that the Khiops source version matches the input tag
+      shell: bash
+      run: |
+        SOURCE_VERSION=$(./scripts/khiops-version)
+        if [[ "$SOURCE_VERSION" != "${{ github.ref_name }}" ]]
+        then
+          echo "::error::Incompatible source version '$SOURCE_VERSION' with tag '${{ github.ref_name }}'. Modify it in src/Learning/KWUtils/KWKhiopsVersion.h."
```

Comment:
The "it" in "Modify it" refers to the Khiops version. However, in order for this to be clear, IMHO we need to rephrase this a little bit: either "Modify source version in src/..", or "Incompatible tag ... with source version... Modify it in src/..."

Reply:
Ok I modified it.